.github/workflows/integration_test.yaml

```diff
@@ -2,12 +2,32 @@ name: Integration tests
 
 on:
   pull_request:
-  schedule:
-    # Trigger at 6:00 AM and 6:00 PM UTC
-    - cron: "0 6,18 * * *"
 
 jobs:
+  check-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      run_tests: ${{ steps.set-output.outputs.run_tests }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: tj-actions/changed-files@v44
```

Comment:
I'd pin to a mayor.minor.patch version instead. I didn't know this exist, btw, I coded myself the logic for the operator workflows :/